hr_timesheet_employee_cost_contract/tests/test_hr_timesheet_employee_cost_contract.py

```diff
@@ -104,24 +104,53 @@ def test_annual_avg(self):
                     'name': 'Employee Contract #1',
                     'wage': 3000.0,
                     'state': 'open',
-                    'date_start': date(2020, 1, 1),
+                    'date_start': date(self.today.year, 1, 1),
                 }),
                 (0, 0, {
                     'name': 'Employee Contract #2',
                     'wage': 1000.0,
                     'state': 'open',
-                    'date_start': date(2020, 1, 1),
+                    'date_start': date(self.today.year, 1, 1),
                 }),
             ],
         })
-        as_of_day = date(2020, 12, 1)
+        as_of_day = date(self.today.year, 12, 1)
         contracts = employee._get_timesheet_cost_contracts(as_of_day)
         self.assertEqual(len(contracts), 2)
         average_hourly_cost = contracts._compute_average_hourly_cost(
             'annual_avg',
             self.eur,
             as_of_day
         )
+        # Employee Contract #1 2021
+        # Jan 3000 / 168 = 17.85
+        # Feb 3000 / 160 = 18.75
+        # Mar 3000 / 184 = 16.30
+        # Apr 3000 / 176 = 17.04
+        # May 3000 / 168 = 17.85
+        # Jun 3000 / 176 = 17.04
+        # Jul 3000 / 176 = 17.04
+        # Aug 3000 / 176 = 17.04
+        # Sep 3000 / 176 = 17.04
+        # Oct 3000 / 168 = 17.85
+        # Now 3000 / 176 = 17.04
+        # Dec 3000 / 184 = 16.30
+        # Average          17.26
+        # Employee Contract #2 2021
+        # Jan 1000 / 168 =  5.95
+        # Feb 1000 / 160 =  6.25
+        # Mar 1000 / 184 =  5.43
+        # Apr 1000 / 176 =  5.68
+        # May 1000 / 168 =  5.95
+        # Jun 1000 / 176 =  5.68
+        # Jul 1000 / 176 =  5.68
+        # Aug 1000 / 176 =  5.68
+        # Sep 1000 / 176 =  5.68
+        # Oct 1000 / 168 =  5.95
+        # Now 1000 / 176 =  5.68
+        # Dec 1000 / 184 =  5.43
+        # Average        =  5.75
+        # Both contracts = 11.50
         self.assertEqual(average_hourly_cost, 7.53)
```

Comment:
@alexey-pelykh I'm trying to understand how you generated your 7.53 here in order to fix the test. The current 11.5 the code returns seem correct to me, do you remember why it's not?

Reply:
Because average of two contracts would be 2000 and divided by 11 working month instead of 12 would give approx 7.53, so your calculation is correct and if you'd fix my another mistake, it should've been something close to 11.50

Thanks for noticing the issue :)